theme/templates/resource-landing-page/manage-access.html

```diff
@@ -155,33 +155,35 @@ <h3 class="panel-title">Sharing status</h3>
                                     <div class="btn-group" role="group">
                                         {# -------- PUBLIC -------- #}
                                         <button
-                                            :disabled="!canBePublicDiscoverable || resAccess.isPublic || isProcessingAccess"
+                                            :disabled="!canBePublicDiscoverable || !canChangeResourceFlags || isProcessingAccess"
                                             @click="setResourceAccess('make_public')"
                                             :class="{active: resAccess.isPublic}"
                                             class="btn btn-default"
-                                            id="btn-public" data-toggle="tooltip" data-placement="auto"
-                                            title='Can be viewed and downloaded by anyone.' type="button">
+                                            id="btn-public" data-toggle="tooltip" data-placement="auto" 
+                                            :title="!canChangeResourceFlags ? this.accessDeniedTitle : 'Can be viewed and downloaded by anyone'" 
```

Comment:
Should we add a period (.) to this tooltip message to match with the other two tooltip messages?

Reply:
Good idea, thanks. Just added!